operators/README.md

```diff
@@ -56,8 +56,10 @@ A specific plugin configuration:
 
 ```jsx
 ...
+import MyOrgFooter from '@myorg/custom-footer'
```

Comment:
I think this should make it more clear that `MyOrgFooter` is a component that exists somewhere else and isn't a magic name